kratos/python/add_accessors_to_python.cpp

```diff
@@ -22,28 +22,21 @@
 #include "includes/accessor.h"
 #include "includes/properties.h"
 
+using AcccessorBindType = std::unique_ptr<Kratos::Accessor>;
+
+PYBIND11_MAKE_OPAQUE(AcccessorBindType);
+
```

Comment:
Shouldn't we define this within the `Kratos::Python` namespace for the case in which we export a custom accessor from a property?

Reply:
No, it needs no to be defined outisde the Kratos namespace.